client/app/bundles/course/assessment/submission/submission.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+
+import React from 'react';
+import { render } from 'react-dom';
+import { Router } from 'react-router';
+import history from 'lib/history';
+import ProviderWrapper from 'lib/components/ProviderWrapper';
+
+import store from './store';
+import routes from './routes';
+
+$(document).ready(() => {
+  const mountNode = document.getElementById('submission-edit');
+
+  if (mountNode) {
+    render(
+      <ProviderWrapper store={store}>
+        <Router history={history}>{routes}</Router>
```

Comment:
Right now `SubmissionEditLayout` is just a wrapper. Perhaps merge the contents of `routes.jsx` into `SubmissionEditLayout` and render `SubmissionEditLayout` here instead of `{routes}`?